src/viking.js

```diff
@@ -1,14 +1,56 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength
+  };
+
+  receiveDamage(damage) {
+    this.health -= damage
+  };
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (Viking === "alive"){
+      return (`${this.name} has receive ${this.damage}`);
+    } else {
+      return (`${this.name} has receive died in combat`)
+    }
+  };
+
+  battleCry(){
+    return ('Odin Owns You All!')
+  }
+}
+
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier { 
+  
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (Saxon === 'alive' ) {
+      return (`A Saxon has received ${this.damage} points of damage`)
```

Comment:
here and also in the Viking class it must be just "damage" because you give it as a parameter to the function (instead of creating it in the class in the constructor). So in code, this one you could have solved like this:
```
  receiveDamage(damage) {
    this.health -= damage;
    if (this.health > 0) {
      return (`A Saxon has received ${damage} points of damage`)
    } else {
      return (`A Saxon has died in combat`)
    }
  }
}
```

Reply:
If you have any questions about this, feel free to ask us!